### DISCUSSION TITLE
Discussions are open!

### DISCUSSION THREAD

**Discussion by sandrotosi** [puddletag/puddletag#572] [Ideas]
Just enabled this github options, so people can ask questions here instead of opening an issue (when appropriate, for bug reports/feature requests please do use issues)

**Comment by catmycode** [comment]
Hello!
Not sure if I am supposed to ask this kind of questions here but I'll give it a try since I understand that the puddletag forum at sourceforge is inactive since 2018.

I started using puddletag a while ago (great app by the way), and came across the following problem. I want to rename my files as follows: 01 - Pavlov's Dog - Late November

The most close way I found is with the following naming pattern: %__counter% - %artist% - %title%
However, I want all numbers have two digits. So, how to I make __counter put a leading zero for numbers 1-9? I tried many variants like $num(%__counter%,2) etc. with no success. Apparently I do not understand its use very well and it seems that there isn't any documentation on how to use it properly.

Thanks!

**Reply by sandrotosi** [reply]
please log a separate issue for your request -- thanks
